vertical-pod-autoscaler/go.mod

```diff
@@ -5,6 +5,7 @@ go 1.12
 require (
 	github.com/beorn7/perks v1.0.1 // indirect
 	github.com/golang/mock v1.3.1
+	github.com/google/uuid v1.1.1
```

Comment:
Did you run go mod vendor? Not sure if this should cause a change in vendor forlder

Reply:
I realized that uuid was used during implementation, but I removed it from the final version...
I'm removing go.mod/go.sum changes.

Also, it seems that I messed some vendor update in one of the previous commit :/ The change is unrelated to current PR, so I will fix it in separate PR.